tls/_constructs.py

```diff
@@ -109,6 +109,32 @@
     "truncated_hmac"
 )
 
+SHA1Hash = Bytes("sha1_hash", 20)
+
+DistinguishedName = PrefixedBytes(
+    "DistinguishedName",
+    SizeWithin(UBInt16("DistinguishedName_length"),
```

Comment:
Hm, I've been using "length" in `Size*` instead of "construct_name_length". I wonder if one convention is better than the other. (No action needed from your end, just a note to self)